packages/lib/src/global.d.ts

```
@@ -0,0 +1,3 @@
+interface window {
+  // Add any custom window properties here if needed
+}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)